rune/libenclave/internal/runtime/pal/skeleton/tls-server.c

```diff
@@ -0,0 +1,76 @@
+#include <netinet/in.h>
+#include <sys/un.h>
+#include <unistd.h>
+#include <stdio.h>
+#include "tls-server.h"
+#include "sgx_urts.h"
+
+#ifdef TLS_SERVER
+bool tls_server = true;
+#else
+bool tls_server = false;
+#endif
+
+extern int ra_tls_server_startup(sgx_enclave_id_t id, int sockfd);
+
+int ra_tls_server(void)
+{
+	printf("    - Welcome to tls server\n");
+
+	int updated = 0;
+	sgx_enclave_id_t eid;
+	sgx_launch_token_t t;
+	memset(t, 0, sizeof(t));
+
+	int ret = sgx_create_enclave(ENCLAVE_FILENAME, 1, &t, &updated, &eid,
```

Comment:
It is unwise to unconditionally set the debug flag. I will create another issue to globally address this sort of problem.